fighters/common/src/function_hooks/change_status.rs

```diff
@@ -94,6 +94,16 @@ unsafe fn change_status_request_from_script_hook(boma: &mut BattleObjectModuleAc
                 }
             }
         }
+        // Checks whether you have landed on the first frame of non-tumble knockback
```

Comment:
why not have it check in the DAMAGE_AIR pre status script if the previous status was crouching
There are almost certainly scenarios where you CC a move and get lifted up for more than a frame?

Reply:
> why not have it check in the DAMAGE_AIR pre status script if the previous status was crouching There are almost certainly scenarios where you CC a move and get lifted up for more than a frame?

u right, just pushed an update